### PR TITLE
feature(keycloak): Add missing limits

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ OpenShift includes the ability for a service account to act as a limited OAuthCl
 for more details). Let's create the service account with the correct redirect URIs enabled:
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/support/serviceaccount-as-oauthclient-restricted.yml
+$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/support/serviceaccount-as-oauthclient-restricted.ymlgit dgit
```
Comment (jimmidyson) [inline README.md]: typo/copy-paste error: `ymlgit dgit`

Comment (syndesis-dev) [reply README.md]: thanks !

Comment (rhuss) [reply README.md]: oops. the curse of testing ;-)

**Comment by rhuss** [comment]
Going to rebase now

**Comment by rhuss** [comment]
@jimmidyson pr would be ready to merge now

**Pull request merged**